src/display/lv_display.c

```diff
@@ -412,6 +412,10 @@ void lv_display_set_buffers(lv_display_t * disp, void * buf1, void * buf2, uint3
         return;
     }
 
+    if(LV_DISPLAY_RENDER_MODE_PARTIAL == render_mode) {
```

Comment:
move up to 408 and change 408 to else if

Reply:
Done